spec/models/post_mover_spec.rb

```diff
@@ -47,16 +55,21 @@
     context 'success' do
 
       it "enqueues a job to notify users" do
-        topic.stubs(:add_moderator_post)
+        Jobs.stubs(:enqueue)
```

Comment:
But the whole point of this test is to check that another job (`:notify_moved_posts`) has been enqueued, so any other jobs should be ignored. That's basically the same the old stub did but I had to change it because the new code ensures that `add_moderator_post` returns an actual post and uses the return value, so leaving that stub there would crash. So I just stubbed whatever jobs are enqueued from inside `add_moderator_post`.